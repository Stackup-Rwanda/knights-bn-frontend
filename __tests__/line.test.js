import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Line from '../src/components/line';

describe('===============> Line <=============', () => {
  afterEach(cleanup);
  it('renders without crushing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Line />, div);
  });
  it('renders', () => {
    const { asFragment } = render(<Line className="Line" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
