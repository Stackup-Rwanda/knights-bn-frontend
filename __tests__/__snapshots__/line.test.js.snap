// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`===============> Line <============= renders 1`] = `
<DocumentFragment>
  <svg
    class="Line"
    viewBox="0 1 120.5 1"
  >
    <path
      d="M 0 1 L 120.5 1"
      fill="transparent"
      id="Line"
      linecap="square"
      linejoin="miter"
      miterlimit="10"
      rendering="auto"
      stroke="rgba(231,231,231,1)"
      width="1px"
    />
  </svg>
</DocumentFragment>
`;
